packages/commonwealth/shared/utils.ts

```diff
@@ -51,9 +52,7 @@ export const getThreadUrl = (
   const tTitle = thread.title ? `-${slugify(thread.title)}` : '';
   const cId = comment ? `?comment=${comment}` : '';
 
-  return process.env.NODE_ENV === 'production'
-    ? `https://commonwealth.im/${aId}/discussion/${tId}${tTitle.toLowerCase()}${cId}`
-    : `http://localhost:8080/${aId}/discussion/${tId}${tTitle.toLowerCase()}${cId}`;
+  return `${SERVER_URL}/${aId}/discussion/${tId}${tTitle.toLowerCase()}${cId}`;
```

Comment:
We shouldn't use environment variables for anything in `shared` since these functions could be imported on the client where the environment variable would not be available and where we should not be importing from `config.ts`. If we want to fix these hardcoded urls then we should instead modify the function to add a URL parameter so that we can set it differently on the client and in the server based on available variables.